lib/toolbar.coffee

```diff
@@ -34,8 +34,8 @@ module.exports =
       default: '24px'
       enum: ['16px', '24px', '32px']
 
-  prependButton: (icon, callback, tooltip = null, iconset = null) ->
-    button = new ToolbarButtonView icon, callback, tooltip, iconset
+  prependButton: (icon, callback, tooltip=null, iconset=null, data=null) ->
```

Comment:
You can always [rebase](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) your original PR.